tests/Panel/Ui/Buttons/LanguagesButtonTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Kirby\Panel\Ui\Buttons;
+
+use Kirby\Panel\Areas\AreaTestCase;
+
+/**
+ * @coversDefaultClass \Kirby\Panel\Ui\Buttons\LanguagesButton
+ * @covers ::__construct
+ */
+class LanguagesButtonTest extends AreaTestCase
+{
+	/**
+	 * @covers ::render
+	 */
+	public function testSingleLang()
+	{
+		$button = new LanguagesButton();
+		$this->assertNull($button->render());
+	}
+
+	/**
+	 * @covers ::option
+	 * @covers ::options
+	 * @covers ::props
+	 * @covers ::render
```

Comment:
Really struggling to understand why this doesn't result in coverage for option and options methods in the report.